app/components/ilios-course-list.js

```diff
@@ -4,11 +4,23 @@ import DS from 'ember-data';
 const { computed, inject, RSVP, ObjectProxy, Component } = Ember;
 const { service } = inject;
 const { PromiseObject } = DS;
+const { collect, sort } = computed;
 
 const CourseProxy = ObjectProxy.extend({
   content: null,
   currentUser: null,
   showRemoveConfirmation: false,
+  status: computed('content.isPublished', 'content.isScheduled', function(){
+    let course = this.get('content');
+    if (course.get('isScheduled')) {
+      return 's';
```

Comment:
since this is supposed to be an alphabetical sorting, it would be better if we used the translatable values for 'scheduled', 'published' and 'not published here'.